packages/expo-cli/src/commands/build/ios/appleApi/authenticate.js

```diff
@@ -52,22 +53,32 @@ async function _promptForAppleId({ appleId }) {
 
   log(wrap(chalk.bold('The password is only used to authenticate with Apple and never stored.')));
 
-  const appleIdQuestions = [
-    {
-      type: 'input',
-      name: 'appleId',
-      message: `Apple ID:`,
-      validate: nonEmptyInput,
-      when: () => !appleId,
-    },
+  if (!appleId) {
```

Comment:
Why? Note there is `when: () => !appleId,` in the question object, so this `if` is redundant.